apps/studio/sentry.client.config.ts

```diff
@@ -39,15 +34,9 @@ Sentry.init({
     'The provided `href` (/org/[slug]/team) value is missing query values (slug)',
     'The provided `href` (/org/[slug]/billing) value is missing query values (slug)',
     'The provided `href` (/org/[slug]/invoices) value is missing query values (slug)',
-    // [Joshen] Seems to be from hcaptcha
-    "undefined is not an object (evaluating 'n.chat.setReady')",
-    "undefined is not an object (evaluating 'i.chat.setReady')",
   ],
-  denyUrls: [
-    // [Joshen] We're getting a huge influx of events from this particular source
-    'injectCoinbaseWalletDappProvider.js',
-  ],
-  beforeSend: (event) => filterConsoleErrors(event),
+  // Allow for exception captures originating only from our own code.
+  allowUrls: ['app:///_next/'],
```

Comment:
All errors that are thrown from our own code will *always* start with this prefix, as it's enforced by `@sentry/nextjs` SDK itself.

Reply:
It can be expanded further once we have a smooth integration and all our own issues are resolved.